src/LongRange/LRCoulombSingleton.cpp

```diff
@@ -271,11 +275,54 @@ createSpline4RbyVs_temp(LRHandlerBase* aLR, T rcut, LinearGrid<T>* agrid)
   return V0;
 }
 
+template<typename T>
+OneDimCubicSpline<T>*  
+createSpline4RbyVsDeriv_temp(LRHandlerBase* aLR, T rcut, LinearGrid<T>* agrid)
+{
+  typedef OneDimCubicSpline<T> func_type;
+  if(agrid==0)
```

Comment:
We should use nullptr for comparisons and assignments to pointers, rather than 0 or NULL. It makes the code clearer that a pointer is involved, and it can do better type checking.